packages/firestore-types/index.d.ts

```diff
@@ -20,21 +20,45 @@ import { FirebaseApp, FirebaseNamespace } from '@firebase/app-types';
  * Document data (for use with `DocumentReference.set()`) consists of fields
  * mapped to values.
  */
-export type DocumentData = { [field: string]: any };
+export type DocumentData = {
+  [field: string]: any;
```

Comment:
Oh, I see. It looks like if I remove the semicolon, prettier adds it back, but if I completely undo this change (remove the semicolon _and_ put this back to being one line) then prettier leaves it as-is...  Maybe you ended up formatting with a "wrong" version and then the "right" version, which had a different net effect than just formatting with the "right" version or something?  I don't know...  Anyway, I'm not inclined to worry about it for this PR...

Reply:
Hmm, this sounds like it could be the cause. I first formatted using `clang-format`, which resulted in a pretty different formatting, and I guess `prettier` couldn't restore it successfully.